Lib/test/test_argparse.py

```diff
@@ -425,7 +431,9 @@ class TestOptionalsAlternatePrefixChars(ParserTestCase):
         Sig('::bar'),
         Sig('/baz', action='store_const', const=42),
     ]
-    failures = ['--bar', '-fbar', '-b B', 'B', '-f', '--bar B', '-baz', '-h', '--help', '+h', '::help', '/help']
+    failures = [
```

Comment:
I think that is better use the first line to put values, and not start on new line. e.g.:

failures = [ '--bar', '-fbar', '-b B', 'B', '-f', '--bar B',
                 '-baz', '-h', '--help', '+h', '::help', '/help']